app/views/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>
```

Comment:
This file should be in the spots folder - will be corrected by Malcolm in another pull request